chia/simulator/block_tools.py

```diff
@@ -774,8 +774,6 @@ def get_consecutive_blocks(
                         else:
                             if guarantee_transaction_block:
                                 continue
-                        if pending_ses:
-                            pending_ses = False
```

Comment:
apparently the person who wrote this code though we needed to clear `pending_ses` here. If we don't, I think it would be appropriate to leave a comment explaining why that's wrong

Reply:
yeah i just changed what the pending_ses variable did, as it was literally useless before

Reply:
by useless i mean it did absolutely nothing, so any past use is irrelevant.

Reply:
would you mind adding a comment where `pending_ses` is declared, describing what its new meaning is?

Reply:
added